project-reports/2023/2023-Q3-Hyperledger-FireFly.md

```diff
@@ -0,0 +1,124 @@
+
+We have onboarded 3 new maintainers to the project and one maintainer has transitioned to a new company. These new maintainers include Matthew Whitehead & Chengxuan Xing from Kaleido and Dzianis Andreyenka from Instincttools. Vinod Damle has moved to Fidelity Investments and will continue as a maintainer.
+
+The Hyperledger FireFly team has been working closely with developers at OneOf over the past year as they built a new FireFly blockchain connector for the Tezos blockchain. They also demod this new blockchain connector at a recent FireFly Community Call. This codebase is in the process of being open sourced and donated to the Hyperledger FireFly project.
+
+Community adoption updates:
+
+- The DFCRC and Reserve Bank of Australia released the [results of their CBDC Pilot](https://www.rba.gov.au/payments-and-infrastructure/central-bank-digital-currency/pdf/australian-cbdc-pilot-for-digital-finance-innovation-project-report.pdf) which leveraged Hyperledger FireFly. Some use cases tested in the pilot include:
+  - A wholesale car dealer generated a tokenized invoice representing a claim on a wholesale buyer.
+  - Offline customer-to-merchant transactions using NFC-enabled smart cards and pilot CBDC between RMIT University and Southern Cross University
+  - Two banks trading tokenized debt securities in wholesale markets
+
+Community members have given talks featuring Hyperledger FireFly at a number of events including:
+
+- Consensus 2023: [The CBDC Tech Stack: How One Ethereum Firm is Working with Central Banks](https://www.youtube.com/watch?v=L1MeqCRfQDw&ab_channel=Kaleido)
+- Hyperledger Workshop: [Hyperledger FireFly and DIDs](https://www.youtube.com/watch?v=ICp6AX9zsW8&ab_channel=HyperledgerFoundation)
+- Webinar: [Launch Enterprise Scale NFT Experiences That Customer Love](https://www.youtube.com/watch?v=YrrSp2SKcfQ&ab_channel=Kaleido)
+- EthCC: [State of Enterprise Blockchain](https://www.youtube.com/watch?v=1XruoWm9cF4)
+- Point Zero Forum: [SME Financing via Programmable Money and Tokenization hosted by Bank of Thailand](https://www.pointzeroforum.com/insights/sme%20financing%20via%20programmable%20money%20and%20tokenisation?hsLang=en)
+
+Contributor activity updates:
+
+The star activity for the FireFly repository has continued to grow from 339 end of last quarter to 398 end of this quarter. Discord activity remains consistent with ~45 DAU.
+
+![alt_text](images/ffq3image1.png "image_tooltip")
+
+Contributors are active in the project and since June there were 14 new contributors who performed at least one commit.
+
+There are currently 748k lines of code for Hyperledger FireFly across 19 repositories, with a total of 7.7k commits to date.
+
+# Questions/Issues for the TOC
+
+Given the maturity of the project, and the recent changes in maintainership, FireFly meets the required and recommended criteria for Incubation Exit and will be submitting a separate proposal to the TOC to become a Graduated Project.
+
+# Releases
```

Comment:
Are there any LTS releases yet?

Reply:
Good question. We have not designated any releases as and LTS release yet, but it is something that we are actively talking about as a community. We need to define the cadence for LTS releases, how long they will be supported for etc. But it is something that we would like to do in the near future.